File: Gabriel's examples/GabrielFedorowicz_Example2.dart
// Control Structure Example: Switch Fall-Through (Dart)
void main() {
  var value = 1;

  switch (value) {
    case 1:
      print("Dart: matched 1");
    // required — no fall-through allowed. Must break of use continue keyword.

    case 2:
      print("Dart: matched 2");
      break;

    default:
      print("Dart: default");
  }
}

File: Gabriel's examples/GabrielFedorowicz_Example2.ts
// Control Structure Example: Switch Fall-Through (TypeScript)
let value: number = 1;

switch (value) {
  case 1:
    console.log("TS: matched 1");
    // No break here → falls through to the next case

  case 2:
    console.log("TS: also reached case 2 due to fall-through");
    break;

  default:
    console.log("TS: default");
}